lib/php/libsdk/SDK/Config.php

```diff
@@ -60,14 +60,13 @@ public static function getCurrentArchName() : string
 			}
 
 			exec("cl.exe /? 2>&1", $a, $status);
-			if ($status) {
-				throw new Exception("Couldn't execute cl.exe.");
```

Comment:
Perhaps it would make sense to keep the exception here, but change the command to `where cl.exe`. In this case only the status matters. If this passed, one can proceed further and exec the actual `cl.exe` to parse the output.

Reply:
hmm... yep, this could be much better. I will make some tests tomorrow.